code/modules/clothing/spacesuits/_spacesuits.dm

```diff
@@ -44,4 +45,5 @@
 	strip_delay = 80
 	equip_delay_other = 80
 	resistance_flags = NONE
+	rad_flags = RAD_PROTECT_CONTENTS | RAD_NO_CONTAMINATE //rated for cosmic radation :honk:

```

Comment:
Not an expert in that field of physics (the game is very black and white on how it works anyway), yet this will heavily ease how to deal with radiation hazards ingame, and should be limited to engi, CE, ERT, deathsquaddies et similar hardsuits, otherwise making rad suits outdated and redunant.
This is my peeve, the added rad flags to several storage objects are fine due to decontamination being quite an hassle to perform in the game.